dotCMS/src/main/java/com/dotmarketing/portlets/htmlpageasset/business/render/HTMLPageAssetRenderedAPIImpl.java

```diff
@@ -367,21 +367,45 @@ private Host resolveSite(final PageContext context, final HttpServletRequest req
 
         final User user = context.getUser();
         final PageMode mode = context.getPageMode();
-        
+
+        Host site = this.getHostFromRequest(request, user, mode);
+
+        if (site == null) {
+            site = this.getHostFromSession(request, user, mode);
+        }
+        System.out.println("site = " + site);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [System.out.println is used](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4718001)